docker/dev.yml

```diff
@@ -33,18 +33,15 @@ x-robotoff-dev: &robotoff-dev
 services:
   api:
     <<: *robotoff-dev
-  worker_high:
+  worker_high_1:
```

Comment:
you are defining a service which is completely different from worker_high.

I don't think it's what you intended to do…

Reply:
oups sorry this one, was unintentional (I did not see you modified docker-compose.yml)